Update Powershell to install module

montereyharris opened the pull request:
Change parameter from -requiredverison to -MinimumVersion. This guarantees user will install latest version but no older than 2.6.0. Module is now on version 4.1.1 so installing 2.6.0 would not be ideal for the user.

azurecla commented:
Hi __@ruchic__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

montereyharris committed:
Update Powershell to install module

Change parameter from -requiredverison to -MinimumVersion. This guarantees user will install latest version but no older than 2.6.0
